rllib/agents/slateq/slateq_torch_policy.py

```diff
@@ -154,7 +155,7 @@ def build_slateq_losses(
 
     clicked = torch.sum(click_indicator, dim=1)
     mask_clicked_slates = clicked > 0
-    clicked_indices = torch.arange(batch_size)
+    clicked_indices = torch.arange(batch_size).to(policy.device)
```

Comment:
Here, this seems (almost) correct.

```
clicked_indices = torch.arange(batch_size).to(clicked.device). # <- some tensor that we know is already on one of the GPU.
```

Reply:
yep. this was actually how I did it originally. :)
updated.